Package.json error during webpack bundling

I see that there have been [recent changes](https://github.com/nats-io/nats.ts/commit/57989ad5e1245c258c47489b591e35b10b5f3806) in the resolution of the package version. Unfortunately, it appears that the current method of resolving the package version presents issues when attempting to compile using Webpack. When bundling a package that uses `ts-nats`, there is a warning from Webpack:

```
arbitrary-package:     WARNING in /arbitrary-package/node_modules/ts-nats/lib/nats.js 41:18-34
arbitrary-package:     Critical dependency: the request of a dependency is an expression
```

At runtime, for the bundled package, this presents the following: 
```
Error: Cannot find module '//../../package.json'
    at r (/arbitrary-package/index.js:369:282738)
    at Object.<anonymous> (/arbitrary-package/index.js:83:36019)
    at Object.<anonymous> (/arbitrary-package/index.js:83:39628)
    at r (/arbitrary-package/index.js:1:158)
    at Object.<anonymous> (/arbitrary-package/index.js:369:250401)
    at r (/arbitrary-package/index.js:1:158)
    at Object.<anonymous> (/arbitrary-package/index.js:369:249991)
    at r (/arbitrary-package/index.js:1:158)
    at Object.<anonymous> (/arbitrary-package/index.js:369:249517)
    at r (/arbitrary-package/index.js:1:158)
    at Object.<anonymous> (/arbitrary-package/index.js:369:249004)
    at r (/arbitrary-package/index.js:1:158)
    at Object.<anonymous> (/arbitrary-package/index.js:170:257317)
    at r (/arbitrary-package/index.js:1:158)
    at Object.<anonymous> (/arbitrary-package/index.js:362:203591)
    at r (/arbitrary-package/index.js:1:158) {
  code: 'MODULE_NOT_FOUND'
}
```

I think the issue may be an extraneous `/` in the path for the package.json, in [nats.js](https://github.com/nats-io/nats.js/blob/master/lib/nats.js) for example, it appears to use the path `../package.json` instead of the `/../package.json` path used in this library.

--- a/package.json
+++ b/package.json
@@ -35,8 +35,8 @@
     "cover:html": "nyc report --reporter=html && open coverage/index.html",
     "cover:coveralls": "nyc report --reporter=lcovonly && cat ./coverage/lcov.info | coveralls",
     "pack": "npm run build && npm pack",
-    "debugtest": "tsc && node node_modules/.bin/ava --verbose -T 6500000 --match",
-    "test": "tsc && nyc ava --verbose -T 15000",
+    "debugtest": "tsc && cp package.json lib/package.json && node node_modules/.bin/ava --verbose -T 6500000 --match",
+    "test": "tsc && cp package.json lib/package.json && nyc ava --verbose -T 15000",
     "doc": "node_modules/.bin/typedoc --options ./typedocconfig.ts && touch ./docs/.nojekyll"
   },
   "engines": {

--- a/src/nats.ts
+++ b/src/nats.ts
@@ -34,14 +34,8 @@ import {existsSync} from "fs";
 
 export {ErrorCode, NatsError}
 
-// locate our package.json
-let pkgFile = __dirname + '/../package.json';
-if (!existsSync(pkgFile)) {
-    // tests will find it here
-    pkgFile = __dirname + '/../../package.json';
-}
 /** Version of the ts-nats library */
-export const VERSION = require(pkgFile).version;
+export const VERSION = process.env.npm_package_version ?? require('../package.json').version;
 
 /**
  * @hidden

--- a/src/nats.ts
+++ b/src/nats.ts
@@ -35,7 +35,7 @@ import {existsSync} from "fs";
 export {ErrorCode, NatsError}
 
 /** Version of the ts-nats library */
-export const VERSION = process.env.npm_package_version ?? require('../package.json').version;
+export const VERSION = require('../package.json').version;
 
 /**
  * @hidden

